content/applications/hr/appraisals.rst

```diff
@@ -0,0 +1,484 @@
+scheduled in one of two ways, either from the *Appraisals* application dashboard, or from an
+individual appraisal card.
+
+First, navigate to the main *Appraisals* application dashboard by going to
+:menuselection:`Appraisals application --> Appraisals`. Click on the activity icon beneath the
+appraisal date on the desired appraisal card and an activity summary pop-up appears. Click
+:guilabel:`+ Schedule an activity` and a blank activity form appears. Select :guilabel:`Meeting` for
+the :guilabel:`Activity Type` from the drop-down menu. The form changes so only the
+:guilabel:`Activity Type` and :guilabel:`Summary` fields appear. If scheduling an activity other
+than a meeting (such as a phone call or a to-do), the fields that appear on the schedule activity
+form changes.
+
+.. note::
+   The activity icon may appear as a number of different icons, depending on what (if any) scheduled
+   activities are in place, and their due dates. If no activities are scheduled, the icon is a gray
+   clock. For example, the icon may appear as a phone (for a phone call), a cluster of people (for a
+   meeting), a Gantt chart (for an appraisal form to be filled out), among others. The color of the
+   icon indicates the status; a green icon means an activity is scheduled in the future, a red icon
+   means the activity is past due. For more details on activities, refer to the
+   :doc:`../productivity/discuss/overview/plan_activities` document
+
+   .. image:: appraisals/activity-icons.png
+      :align: center
+      :alt: The appraisal cards with the various activity icons highlighted.
+
+Enter a brief summary in the field, such as `Annual Appraisal for (Employee)`. Next, click the
+:guilabel:`Open Calendar` button and the calendar loads. Navigate to and double click on the date
+and time for the meeting, and a :guilabel:`New Event` form pops up. Make any modifications desired
+to the form, such as the start and end times, or the name of the meeting. Add the person being
+appraised in the :guilabel:`Attendees` section, and include anyone ese who should be in the meeting,
+if necessary. To make the meeting a video call instead of an in-person meeting, click :guilabel:`+
+Odoo meeting` and a :guilabel:`Videocall URL` link appears in the field. When done making changes to
+the form, click :guilabel:`Save & Close`. The meeting now appears on the calendar, and the invited
+parties are informed via email.
+
+.. image:: appraisals/meeting.png
+  :align: center
+  :alt: The meeting form with all information entered for Ronnie Hart's annual appraisal.
+
+The other way to schedule a meeting is to navigate to the individual appraisal form. Navigate to the
+main *Appraisal* application dashboard, then click on an appraisal card. Next, click on the
+:guilabel:`Meeting` smart button and the calendar loads. Follow the same directions above to create
+the meeting.
+
+.. note::
+   If no meetings are scheduled (likely) the :guilabel:`Meeting` smart button will say
```

Comment:
```suggestion
   If no meetings are scheduled the :guilabel:`Meeting` smart button will say
```
I would leave the "likely" off, unless you are going to specifically list what other reasons why a meeting would be listed.